CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 ## Unreleased
 
+## 0.26.0 (October 26, 2023)
+
 Changes:
-* Default `vault` version updated to 1.15.0
-* Default `vault-k8s` version updated to 1.3.0
+* Default `vault` version updated to 1.15.1
+* Default `vault-k8s` version updated to 1.3.1
```

Comment:
Should we add to the changes?
* Default `vault-csi-provider` version updated to 1.4.1

Reply:
Yes thanks for pointing that out!

Reply:
fixed